Compilation fails: Found staticlib `cassandra` instead of rlib or dylib

When trying to build a new project using the simple.rs from the README.md file, the build fails with:

error[E0462]: found staticlib `cassandra` instead of rlib or dylib
 --> src/main.rs:2:5
  |
2 |     extern crate cassandra;
  |     ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: please recompile that crate using --crate-type lib
  = note: crate `cassandra` path #1: /usr/lib64/libcassandra_static.a
  = note: crate `cassandra` path #2: /usr/lib/libcassandra_static.a
  = note: crate `cassandra` path #3: /usr/lib64/libcassandra_static.a
  = note: crate `cassandra` path #4: /usr/lib/libcassandra_static.a

error: aborting due to previous error

Based on [https://github.com/tokio-rs/tokio-curl/issues/14](url), perhaps the problem is that the crate is called '**cassandra-cpp**' but `extern crate cassandra` is used instead of `extern crate cassandra-cpp`.

--- a/README.md
+++ b/README.md
@@ -43,8 +43,8 @@ Here's a straightforward example found in simple.rs:
 
 ```rust
     #[macro_use(stmt)]
-    extern crate cassandra;
-    use cassandra::*;
+    extern crate cassandra_cpp;
+    use cassandra_cpp::*;
     use std::str::FromStr;
 
 

